src/app/body/common-tools/update-image/update-image.component.ts

```diff
@@ -55,20 +56,40 @@ export class UpdateImageComponent implements OnInit {
     });
   }
 
-  async setProfilePic() {
-    const callable = this.functions.httpsCallable('users/updateProfilePic');
-   
-    await callable({Uid: this.uid, PhotoURL: this.croppedImage, DisplayName: this.displayName, Email: this.email }).subscribe({
-      next: (data) => {
-        console.log("Successful");
-      },
-      error: (error) => {
-        this.errorHandlerService.showError = true;
-        this.errorHandlerService.getErrorCode(this.componentName, "InternalError","Api");
-        console.error(error);
-      },
-      complete: () => console.info('Successful updated image')
-  });
+  setProfilePic() {
+    if (this.displayName == undefined) {
+      console.log("2")
+      const orgDomain = this.backendService.getOrganizationDomain();
+      const callable = this.functions.httpsCallable('organization/updateOrg');
+
+      callable({OrgDomain: orgDomain, PhotoURL: this.croppedImage}).subscribe({
+        next: (data) => {
+          console.log("Successful");
+        },
+        error: (error) => {
+          this.errorHandlerService.showError = true;
+          this.errorHandlerService.getErrorCode(this.componentName, "InternalError", "Api");
+          console.error(error);
+        },
+        complete: () => console.info('Successful updated image')
+      });
```

Comment:
Update org is now already implemented above, so I think it can be removed from here.

Reply:
done